client/scss/layouts/_home.scss

```diff
@@ -76,61 +76,10 @@
       }
     }
   }
-
```

Comment:
this file will change a lot via https://github.com/wagtail/wagtail/pull/8922

I have incorporated most of your clean up @thibaudcolas 

But also includes https://github.com/wagtail/wagtail/issues/8818

Reply:
👍 thank you, I was able to resolve those conflicts.